package.json

```diff
@@ -13,7 +13,9 @@
     "test": "jest --collectCoverage",
     "test-print-config": "jest --showConfig",
     "eslint": "eslint --max-warnings 0 --ext .js,.jsx,.ts,.tsx",
-    "eslint-print-config": "eslint --print-config"
+    "eslint-print-config": "eslint --print-config",
+    "e2e-test": "cypress run --config-file packages/common/test-configs/cypress.json",
+    "e2e-test:open": "cypress open --config-file packages/common/test-configs/cypress.json"
```

Comment:
if `cypress.json` is placed in the same folder we may not need specify `--config-file` param

Reply:
@vidyanambiar a small cleanup suggestion to keep our scripts consistent:
1. Use `-` in favor of `:` delimiter
2. Can we update the E2E scripts to start with `test-e2e` so they appear closer alphabetically to the other test scripts?